modules/virt-creating-nad-localnet-console.adoc

```diff
@@ -0,0 +1,30 @@
+// Module included in the following assemblies:
+//
+// * virt/vm_networking/virt-connecting-vm-to-ovn-secondary-network.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="virt-creating-nad-localnet-console_{context}"]
+= Creating a NAD for localnet topology using the web console
+
+You can create a network attachment definition (NAD) to connect workloads to a physical network by using the {product-title} web console.
+
+.Prerequisites
+* Use nmstate to configure the localnet to OVS bridge mappings.
```

Comment:
Not familiar with this but should nmstate be formatted with backticks? (is it a command?) Backticks are used in [this module](https://70940--ocpdocs-pr.netlify.app/openshift-enterprise/latest/networking/k8s_nmstate/k8s-nmstate-about-the-k8s-nmstate-operator#k8s-nmstate-about-the-k8s-nmstate-operator) ("OpenShift Container Platform uses `nmstate`") so maybe?